start-gateway.sh

```diff
@@ -80,16 +81,16 @@ EOI
         DEFAULT_FREQUENCY_HTTP_URL="https://0.rpc.testnet.amplica.io"
         DEFAULT_PROVIDER_ID="729"
         DEFAULT_PROVIDER_ACCOUNT_SEED_PHRASE="DEFAULT seed phrase needed"
-        DEFAULT_IPFS_VOLUME="/data/ipfs"
+        DEFAULT_IPFS_ENDPOINT="https://ipfs.infura.io:5001"
```

Comment:
I don't think this is what we want and I don't think it works. Shouldn't the default be the internal one? Or perhaps a public one like dweb.link?

```suggestion
        DEFAULT_IPFS_ENDPOINT="https://dweb.link/ipfs"
```